collect_app/src/main/java/org/odk/collect/android/widgets/ItemsWidget.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.odk.collect.android.widgets;
+
+import android.content.Context;
+
+import org.javarosa.core.model.SelectChoice;
+import org.javarosa.form.api.FormEntryPrompt;
+import org.javarosa.xpath.expr.XPathFuncExpr;
+import org.odk.collect.android.external.ExternalDataUtil;
+
+import java.util.List;
+
+public abstract class ItemsWidget extends QuestionWidget {
```

Comment:
Could you add a Javadoc comment (`/** ... */`) to this class describing its purpose?  There should be enough detail in the description so that a person reading it would know whether a class should be derived from `ItemsWidget` or not.

Reply:
Done.